plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -48,6 +62,10 @@ def validate_prepare(self, prepare: Prepare, sender):
 
     @measure_time(MetricsName.BLS_VALIDATE_COMMIT_TIME)
     def validate_commit(self, commit: Commit, sender, pre_prepare: PrePrepare):
+        if f.BLS_MULTI_SIGS.nm in commit:
+            if not list(filter(lambda sig: self._validate_signature(sender, sig, pre_prepare), commit.blsSigs)):
```

Comment:
`filter` here returns all signatures for which validation succeeded, so if it's empty then either none of them succeeded, or there were no sigs at all. So, according to existing logic:
1) We return `CM_BLS_SIG_WRONG` if all sigs are valid
2) We return `CM_BLS_SIG_WRONG` if there are no sigs
3) We don't return  `CM_BLS_SIG_WRONG` if some but not all sigs are invalid
4) We don't return  `CM_BLS_SIG_WRONG` if all sigs are invalid